### PR TITLE
Fix dialog used with empty PSK credentials.

### PR DESCRIPTION
**Pull request by boaks**
Signed-off-by: Achim Kraus <achim.kraus@cloudcoap.net>

### CODE REVIEW

File: cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java [hunk 1]
```diff
@@ -96,10 +100,6 @@ private void secretToView() {
 			formatBase64.setSelected(true);
 			secret = secretConfig.base64;
 			format = "base64";
-		} else {
-			formatText.setSelected(true);
-			secret = "";
-			format = "text";
```
Comment (sophokles73) [inline cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java:102]: how about also fixing the indentation in this method?

Comment (boaks) [reply cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java:102]: Hm, I applied [Eclipse/Californium - eclipse-formatter-profile](https://github.com/eclipse/californium/blob/main/eclipse-formatter-profile.xml). So I'm not sure, what is wrong with the indentation?

Comment (sophokles73) [reply cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java:102]: seems to be a display problem in the GitHub diff viewer. If you expand the code section above your change, then it looks like the indentation were wrong, doesn't it?

Comment (boaks) [reply cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java:102]: Maybe, the main difference is, that beside a minimum (apply the formatter), I don't care too much about the format ;-). 
My firefox doesn't show something, which looks bad for me.
Do you more/less indention? We use tabs.

Comment (sophokles73) [reply cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java:102]: never mind

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**